files/root/.bashrc

```diff
@@ -0,0 +1,29 @@
+# This file is managed by puppet.
+#
+# Everything in this file is lightly adapted from default Debian or Ubuntu
+# .bashrc. It's not meant to be opinionated, we don't keep bikes in the
+# datacenter, so we'll need no sheds. This file exists purely for quality of
+# life in the rare case we login as root. DO NOT PUT ANYTHING IN THIS FILE THAT
+# IS ACTUALLY CRITICAL TO MAKE A SYSTEM WORK. If ANYTHING in this file offends
+# your sensibilities, change it and file a PR.
+
+# If not running interactively, don't do anything
+[ -z "$PS1" ] && return
+
+# color, abbreviation
+COLOR_OPTIONS='--color=auto'
+eval "$(dircolors)"
+alias l='ls'
+alias ls='ls $COLOR_OPTIONS'
+alias ll='ls $COLOR_OPTIONS -l'
+alias la='ls $COLOR_OPTIONS -A'
+alias lla='ls $COLOR_OPTIONS -lA'
```

Comment:
I know you said this was largely copied from debian/ubuntu defaults, so I'm not inclined to do much bikeshedding, but since these are meant to be convenient, I'll suggest `s/-l/-lh/`, since `-h` is almost always preferable for taking information in at a glance.